x-pack/plugins/snapshot_restore/public/application/sections/home/snapshot_list/snapshot_list.tsx

```diff
@@ -124,38 +123,6 @@ export const SnapshotList: React.FunctionComponent<RouteComponentProps<MatchPara
         error={error as Error}
       />
     );
-  } else if (Object.keys(errors).length && repositories.length === 0) {
```

Comment:
Since the repository name is included in the snapshots API response now, we do not fetch the repositories separately and therefore will not reach this scenario. Any errors will be captured if the request errors.